python/lsst/ctrl/bps/wms/panda/conf_example/example_panda_SLAC.yaml

```diff
@@ -0,0 +1,107 @@
+pipelineYaml: "/sdf/home/j/jchiang/PanDA_testing/pipelines/cpBias.yaml"
+#  OR
+#USER qgraphFile: "/path/to/existing/file.qgraph"
+
+#PANDA plugin specific settings:
+idds_server: "https://aipanda015.cern.ch:443/idds"
+placeholderParams: ['qgraphNodeId', 'qgraphId']
+
+#IDF PanDA specific settings:
+#computing_cloud: LSST
+
+#SLAC PanDA specific settings:
+computing_cloud: US
+computeSite: DOMA_LSST_SLAC_TEST
+
+
+
+# At minimum, following group used in bps report and can be
+# used in WMS queries.  Users can use in other things such as "output". 
+operator: spodolsky    # defaults to login on submit machine
+project: dev
+campaign: quick
+
+# Any extra site-specific settings needed for WMS
+#USER computeSite: ncsapool
+
+# Make sure these values correspond to ones in the bin/run_demo.sh's
+# pipetask command line.
+payload:
+  payloadName: pipelines_check    # At minimum, used in bps report
+  runInit: true
+  output: "u/spadolski/bias_12781_panda_test/{timestamp}"
+  outCollection: u/spodolsky/bias_12781_panda_test/{timestamp}
+
+  butlerConfig: /sdf/group/lsst/camera/IandT/repo_gen3/panda_test_repo
+  inCollection: LSSTCam/raw/all,LSSTCam/calib
+  exposureList: "(3020111900015,3020111900016,3020111900017,3020111900018,3020111900019,3020111900020,3020111900021,3020111900022,3020111900023,3020111900024,3020111900025,3020111900026,3020111900027,3020111900028,3020111900029,3020111900030)"
+  dataQuery: "instrument='LSSTCam' and exposure in {exposureList}"
+
+  #USER butlerConfig: ${PIPELINES_CHECK_DIR}/DATA_REPO/butler.yaml
+  #USER inCollection: HSC/calib,HSC/raw/all,refcats
+  #USER dataQuery: exposure=903342 AND detector=10
+
+  payload_folder: payload
+  fileDistributionEndPoint: "/sdf/group/lsst/software/IandT/tmp/{payload_folder}/{uniqProcName}/"
+  runner_command: 'unset PYTHONPATH;export LSST_DB_AUTH=/sdf/home/l/lsstsvc1/.lsst/db-auth.yaml;source /sdf/group/lsst/software/IandT/stack-lsst_distrib-d_2021_08_03/loadLSST.bash;cd ${PILOT_HOME};pwd;ls -a;setup lsst_distrib;env;python3 ${CTRL_BPS_DIR}/python/lsst/ctrl/bps/wms/panda/edgenode/cmd_line_decoder.py _cmd_line_ '
```

Comment:
Where do you document the magic `_cmd_line_` string that appears at the end of this?

Reply:
The question was where it was documented, not what it is.

Reply:
Updated